wolox-training/vs-rails
# Authentication working

## Authentication working

**Summary**
- Added authentication using [Devise](https://github.com/plataformatec/devise)
- The users can sign up, sign in and sign out of the blog
- When you are not signed in you can only see the list of articles, and the specific articles, but you can't create, edir or destroy any content
- If you are signed in you can also see a list of users registered.
- If you sign in while you were trying to create a comment for example, after you sign in you should be redirected to where you were before.

**Test plan**
- Try signing up, and you should be able to create, edit or delete content
- Try seeing the list of users while you are signed in
- Try creating a comment or an article while you are signed out, you should be redirected to the sign in page, and then redirected to where you were before

**Trello card**
https://trello.com/c/cccWJEhg/9-instalar-y-configurar-devise
